recipes/app3.rb

```diff
@@ -155,3 +160,24 @@
   template 'app-nginx.erb'
   cookbook 'osl-app'
 end
+
+#### Nagios Monitoring ####
+
+check_http  = node['osl-nrpe']['check_http']
+check_https = node['osl-nrpe']['check_https']
+
+%w(streamwebs.org
+   streamwebs-staging.osuosl.org).each do |host|
+     nrpe_check "check_http_#{host}" do
+       command "#{node['nrpe']['plugin_dir']}/check_http"
+       parameters "-H #{host} -I #{check_http['ipaddress']} -e #{check_http['codes'].join(',')}"
+       action :add
+     end
+
+     nrpe_check "check_https_#{host}" do
+       command ::File.join(node['nrpe']['plugin_dir'], 'check_http')
+       parameters "--ssl -H #{host} "\
+         "-C #{check_https['cert_days']} -e #{check_https['codes'].join(',')}"
+       action :add
+     end
+   end
```

Comment:
Actually, the preferred method would be to add a databag which does the check. Take a look at the ``data_bags/nagios_services/http_formsender_osuosl_org_server_status.json`` file as an example (although I'd name it something better).